scripts/karma.coffee

```diff
@@ -49,15 +50,13 @@ module.exports = (robot) ->
     seenUsernames = new Set()
     userKarmaExpressions = getUserKarmaExpressions(msg.match)
       .filter((expr) ->
-        # Only keep the first expression seen for per user
-        if expr.username && seenUsernames.has(expr.username)
+        # Unless the user has been already been seen, 
+        # Or unless the user does not exist
+        unless seenUsernames.has(expr.username) or !robot.brain.userForName(expr.username)
```

Comment:
I think this syntax is awkward... should probably be

`if !seenUsernames.has(expr.username) and robot.brain.userForName(expr.username)`